README.md

```diff
@@ -64,32 +64,22 @@ Fine-tune BERT on a small sentiment analysis task using the
 
 ```python
 import keras_nlp
-from tensorflow import keras
 import tensorflow_datasets as tfds
 
-imdb_train, imdb_test = tfds.load(
+dataset = tfds.load(
```

Comment:
Can we keep the test set? I think it's bad practice to fit any model without it.

Reply:
Sure! I like the short punchy version, but not a big deal.